README.md

```diff
@@ -52,7 +52,7 @@ cd test
 ```
 and
 ```bash
-pytest -sv .
+pytest -sv test_xcpp_kernel.py
 ```
```

Comment:
Ahh wait I am confused. So we create build and technically after running make install we have a test dir inside build. Do we also exoect the `test_xcpp_kernel.py` to be present there.

Reply:
@anutosh491 It is on Linux and MacOS, as can be seen from the ci. Its not true for Windows (I found this out fixing the ci for Jupyter Kernel Test on Windows). I plan to fix this when I add instructions for building on Window

Reply:
I am using a linux based device and I don't find this inside my build/test dir. will check again !

Reply:
There is no `test_xcpp_kernel.py` inside build/test. The way I think to check this test is to run this command inside the test dir with all the test files.

Reply:
+1 Atleast the current cmake instruction don't move the `test_xcpp_kernel.py` file into `build/test` for me